.circleci/config.yml

```diff
@@ -130,135 +146,135 @@ jobs:
       - teardown_vm
   'Checks':
     executor:
-      name: amphtml-medium-executor
+      name: docker-medium
     steps:
       - setup_vm
       - install_chrome
       - run:
-          name: 'Checks'
+          name: '⭐ Checks ⭐'
```

Comment:
Supernit - I think `⭐️ Checks` looks better than `⭐️ Checks ⭐️` and is just as easy to scan for

Reply:
I first tried just a star on the left and then went with a star on either side because it was a lot easier to spot on mobile (especially during one-handed use).